### PR TITLE
Issue #13877: SpotBugs: checks a map with containsKey(), before using…

### PR DESCRIPTION
**Pull request by AayushSaini101**
Related: https://github.com/checkstyle/checkstyle/issues/13877

Original Posted: https://github.com/checkstyle/checkstyle/pull/13890

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java [hunk 1]
```diff
@@ -694,9 +694,10 @@ public static String getSinceVersion(String moduleName, DetailNode moduleJavadoc
                                          String propertyName, DetailNode propertyJavadoc)
             throws MacroExecutionException {
         final String sinceVersion;
-        if (SINCE_VERSION_FOR_INHERITED_PROPERTY.containsKey(moduleName + DOT + propertyName)) {
-            sinceVersion = SINCE_VERSION_FOR_INHERITED_PROPERTY
-                    .get(moduleName + DOT + propertyName);
+        final String containsVal = SINCE_VERSION_FOR_INHERITED_PROPERTY
```
Comment (nrmancuso) [inline src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java:697]:
```suggestion
        final String superClassSinceVersion = SINCE_VERSION_FOR_INHERITED_PROPERTY
```

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java (hunk 1 above)
Comment (AayushSaini101) [reply src/main/java/com/puppycrawl/tools/checkstyle/site/SiteUtil.java]: @nrmancuso Done

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLotsofSingleLineComments.java
Comment (nrmancuso) [inline src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLotsofSingleLineComments.java:1]: @AayushSaini101 how is this change related to this issue?

**Review by nrmancuso — CHANGES_REQUESTED** [review]

File: src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLotsofSingleLineComments.java
Comment (AayushSaini101) [reply src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThroughLotsofSingleLineComments.java:1]: @nrmancuso Extra file added removed

**Review by nrmancuso — APPROVED** [review]

**Review by rnveach — APPROVED** [review]
